### ISSUE TITLE
The problem of update Ex and Ey

### ISSUE THREAD

**Issue by sephirothhua** [alpc91/NICE-GAN-pytorch#2]
I read your paper that says "the Ex is fixed and Ey is trained when maximizing L x->y",
However, in the code the Ex and Ey are both update during the D step. I thought they would be updated separately during training. Did I get it wrong？

**Comment by alpc91** [comment]
It means that Ex'weight is only updated by maximizing Lgan y->x  ,  not by  Lgan x->y.
Corresponding to the code,  fake = fake.detach()
